content/before-you-fly/code-of-conduct.md

```diff
@@ -6,46 +6,45 @@ weight: 2
+All pilots using BMSC sites must be members of the Sports Aviation Federation of Australia (SAFA) and undertake to **follow regulations as set out in the SAFA Operations Manual**.
 
-It is a requirement of BMSC membership and use of Club administered flying sites that you agree to observe &amp; abide by the following values set out below. Failure to abide by the ‘Code of Conduct’ could result in suspension of Flying Privileges or escalation to SAFA as deemed necessary at the time by the Committee.
+It is a requirement of BMSC membership and use of Club administered flying sites that you agree to observe and abide by the following values set out below. Failure to abide by the ‘Code of Conduct’ could result in suspension of Flying Privileges or escalation to SAFA as deemed necessary at the time by the Committee.
 
 ## 1. Respect for Club sites.
 
-Flying sites are a precious resource for our sport &amp; our club. Our custodianship of them and ongoing access to them is not guaranteed and is under regular scrutiny. The Mount Blackheath site is our main site, and over the years the club has worked hard to maintain a good relationship with Blue Mountains City Council, and the Landing Paddock landowner. We do not have an alternative landing site for Mt Blackheath, and loss of this would ultimately result in the closure of the site.
+Flying sites are a precious resource for our sport and our club. Our custodianship of them and ongoing access to them is not guaranteed and is under regular scrutiny. The Mount Blackheath site is our main site, and over the years the club has worked hard to maintain a good relationship with Blue Mountains City Council, and the Landing Paddock landowner. We do not have an alternative landing site for Mt Blackheath, and loss of this would ultimately result in the closure of the site.
 
-Please ensure that you access &amp; use them in a manner that enhances our likelihood of keeping them. This includes;
+Please ensure that you access and use flying sites in a manner that enhances our likelihood of keeping them. This includes:
 
-Familiarity with site rules – this is your responsibility. The rules are detailed on our website and site guides, and in some cases on site. If in doubt it is a requirement that you ask a representative of the club for clarity.
-
-Launching/flying/landing only from or within specifically designated areas, in appropriate conditions following approved site procedures.
+- **Familiarity with site rules - this is your responsibility**. The rules are detailed on our website and site guides, and in some cases on site. If in doubt it is a requirement that you ask a representative of the club for clarity.
+- **Launching, flying and landing only from or within specifically designated areas**, in appropriate conditions following approved site procedures.
```

Comment:
Formatting changes only (so website publishes these as list items, and important part bold).